app/src/main/res/values-nl/strings.xml

```diff
@@ -71,11 +71,11 @@
     <string name="invalid_image_path">Ongeldig pad naar afbeelding</string>
     <string name="image_editing_failed">Fout bij bewerken van afbeelding</string>
     <string name="edit_image_with">Afbeelding bewerken met:</string>
-    <string name="no_editor_found">Geen app gevonden voor bewerking</string>
+    <string name="no_editor_found">Geen bewerk-app gevonden</string>
```

Comment:
Yuck

Reply:
This is how we use in the Netherlands. Please, again: start your own Flemish translation if you want to use your style.

Reply:
I'm Dutch.

Reply:
Well, during my study we also covered apps and social media and they kept calling editing apps "bewerk-apps", so I figured that was more correct, but oh well.